yucca/training/lightning_modules/YuccaLightningModule.py

```diff
@@ -26,7 +25,7 @@ def __init__(
         self,
         config: dict,
         learning_rate: float = 1e-3,
-        loss_fn: str = "DiceCE",
+        loss_fn: str = None,
```

Comment:
Why should this be none?

Reply:
Because then we can set the default loss based on the task type (if you scroll down a little bit) so for task types segmentation and classification we use DiceCE as default but for unsupervised it defaults to MSE. 

Ofc, if you set the loss_fn when you call the LightningModule it will just use whatever you specify